src/WrappedMToken.sol

```diff
@@ -530,10 +530,17 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
      */
     function _subtractTotalEarningSupply(uint240 amount_, uint128 currentIndex_) internal {
         unchecked {
+            uint240 totalEarningSupply_ = totalEarningSupply();
+
+            if (amount_ >= totalEarningSupply_) _setTotalEarningSupply(0, 0);
```

Comment:
is the return missing here ?

Reply:
Great catch! I missed the return on line 540 too!